packages/core/index.d.ts

```diff
@@ -193,4 +194,10 @@ export interface ZObject {
     RefreshAuthError: typeof RefreshAuthError;
     ThrottledError: typeof ThrottledError;
   };
+
+  cache: {
+    get: (key: string) => Promise<any>;
+    set: (key: string, value: string, ttl?: integer) => Promise<boolean>;
```

Comment:
Should `value` be `any` instead?

```suggestion
    set: (key: string, value: any, ttl?: integer) => Promise<boolean>;
```

Reply:
🤦 thanks